test/k4FWCoreTest/src/components/ExampleEventHeaderConsumer.cpp

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2014-2023 Key4hep-Project.
+ *
+ * This file is part of Key4hep.
+ * See https://key4hep.github.io/key4hep-doc/ for further info.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "edm4hep/Constants.h"
+#include "edm4hep/EventHeaderCollection.h"
+
+#include "k4FWCore/BaseClass.h"
+
+#include <Gaudi/Property.h>
+#include <GaudiAlg/Consumer.h>
+#include <GaudiKernel/ISvcLocator.h>
+
+#include <fmt/core.h>
+#include <fmt/format.h>
+
+#include <atomic>
+#include <optional>
```

Comment:
Not needed?

Reply:
Right, good catch. I was experimenting with optional inputs in Gaudi::Functional and forgot to remove this.